kalite/distributed/templates/distributed/base.html

```diff
@@ -171,10 +171,10 @@
                               {# Search box for responsive nav #}
                               <div class="visible-xs">
                                   <li id="search_input">
```

Comment:
@jtamiace its okay for the late reply, I will pull the latest develop branch and will update.